src/main/scala/cromwell/binding/formatter/SyntaxFormatter.scala

```diff
@@ -94,31 +98,15 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
     header
   }
 
-  private def formatCommandSection(command: Command, level:Int): String = {
-    val section = s"""${highlighter.section("command")} {
-        |${formatCommand(command, level+1)}
-        |}"""
-    indent(section.stripMargin, level)
-  }
-
-  private def formatCommand(command: Command, level:Int): String = {
-    val abstractCommand = command.parts.map {
-      case x:StringCommandPart => x
-      case x:ParameterCommandPart => formatParameterCommandPart(x)
-    }
-    indent(highlighter.command(command.normalize(abstractCommand.mkString)), 1)
-  }
+  private def formatCommandSection(task: Task, level:Int): String = {
+    val (sdelim: String, edelim: String) =
+      if (task.commandTemplate.collect{case s:StringCommandPart => s.literal}.mkString.contains("}")) ("<<<", ">>>")
```

Comment:
also i think it'd be more clear to take that expression and assign it to a val w/ a meaningful name and then if on that